lighthouse-core/lib/styles-helpers.js

```diff
@@ -96,14 +96,14 @@ function getFormattedStyleRule(content, parsedContent) {
     }, []).join('\n');
   }
 
-  const block = `
-${parsedContent.selector} {
+  const block = `\`\`\`${parsedContent.selector} {
   ${rule}
-}`;
+}\`\`\``;
 
   return {
     styleRule: block.trim(),
-    location: `line: ${startLine}, row: ${start}, col: ${end}`
+    startLine,
+    location: `${start}:${end}`
```

Comment:
we should think about settling on a format for parsed source blocks at some point. ESTree has a semi-standard format (albeit for JS): https://github.com/estree/estree/blob/master/es5.md#node-objects